src/rp2_common/hardware_rtc/rtc.c

```diff
@@ -13,7 +13,7 @@
 
 // Set this when setting an alarm
 static rtc_callback_t _callback = NULL;
-static bool _alarm_repeats = false;
+static int8_t _alarm_repeats = 0;
```

Comment:
prefer to use an enum for 3 repeat types

Reply:
I'll upate it